apps/hoarder/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+services:
+  web:
+    image: ghcr.io/hoarder-app/hoarder:0.16.0
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
```

Comment:
That should be `${APP_DATA_DIR}/data/app:/data` or something similar, the data directory shouldn't be used alone.